### ISSUE TITLE
Reprovisioning with PECL uploadprogress fails the build

### ISSUE THREAD

**Issue by philipnorton42** [hashbangcode/vlad#65]
Steps to reproduce:

1. vagrant up with ```php_pecl_uploadprogress: "y"``` in settings.yml
2. vagrant halt
3. vagrant up --provision

Results in this error and a failed build:

```
TASK: [php | PHP | Install PECL uploadprogress] *******************************
failed: [bohea] => {"changed": true, "cmd": ["pecl", "install", "uploadprogress"], "delta": "0:00:01.527502", "end": "2014-07-08 15:26:45.388105", "rc": 1, "start": "2014-07-08 15:26:43.860603"}
stdout: pecl/uploadprogress is already installed and is the same as the released version 1.0.3.1
install failed

FATAL: all hosts have already failed -- aborting
```

---------------------------------------
- Bitbucket: https://bitbucket.org/philipnorton42/vlad/issue/65
- Originally Reported By: [Dan Bohea](http://bitbucket.org/dbohea)
- Originally Created At: 2014-07-08T17:30:27.258

**Comment by philipnorton42** [comment]
Has anyone seen this issue since I put this fix in place? I'm keen on closing it off if everything is 'tickety-boo'! :)

**Comment by danbohea** [comment]
Not been a problem for me.

**Comment by philipnorton42** [comment]
Setting this as fixed for the upcoming 1.0 release :)
